pkg/ratelimit/ratelimit.go

```diff
@@ -0,0 +1,82 @@
+package ratelimit
+
+import (
+	"time"
+
+	"go.uber.org/ratelimit"
+)
+
+type Event struct {
+	Timestamp time.Time
+	Tag       int
+}
+
+type RateLimiter struct {
+	outputCh     chan Event
+	src          <-chan Event
+	rt           ratelimit.Limiter
+	ch           chan Event
+	doneSender   chan struct{}
+	doneReceiver chan struct{}
+}
+
+func NewWithEPS(eventsPerSecond int64, src <-chan Event) *RateLimiter {
+	rt := newCommon(src)
+	rt.rt = ratelimit.New(int(eventsPerSecond))
+	return rt
+}
+
+func NewUnlimited(src <-chan Event) *RateLimiter {
+	rt := newCommon(src)
+	rt.rt = ratelimit.NewUnlimited()
```

Comment:
considering the uber `ratelimit` package seems to provide all the functionalities we need, can we perhaps simplify our wrapping code?